### PR TITLE
remove some unused fonts and reorder how js loads

### PR DESCRIPTION
**Pull request by anagio**
optimize loading of assets
cleanup fonts

### CODE REVIEW

**Comment by anagio** [comment]
- Theme blue css imported to our main css file
- remove extra market form js
- remove vwo visual website optimizer
- move some js to footer
- add preconnect, dns-prefetch and preload attributes to some header link tags
- replace document.write with better js
- remove unused fonts
- remove fontawesome and replacing with png icons
- add font-display fallback
- load large header images though Imgix auto compressing and sizing

File: _layouts/default.html [hunk 1]
```diff
@@ -53,7 +51,202 @@
 
           {% include_cached footer.html %}
 
-
+
```
Comment (yingzhuchin) [inline _layouts/default.html:54]: This move from `search.html` could be confusing from a contributing standpoint. Can you explain more the enhancement trade-off here?

File: _includes/head.html [hunk 2]
```diff
@@ -24,30 +23,27 @@
 };
 </script>
 
-{% include_cached vwo.html %}
```
Comment (yingzhuchin) [inline _includes/head.html:27]: We can remove this from the `_includes` folder then?

File: _includes/marketo-install.html [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <form id="release-mktoForm_1082_{{ include.uid }}" class="mkto-install-form"></form>
```
Comment (yingzhuchin) [inline _includes/marketo-install.html:1]: Are we using this file anywhere?

Comment (yingzhuchin) [inline _includes/marketo-install.html:1]: nvm, found it!

**Review by yingzhuchin — COMMENTED** [review]
Had a few comments, LGTM otherwise!

File: _includes/head.html (hunk 2 above)
Comment (anagio) [reply _includes/head.html:27]: Yes we can going to remove it

File: _layouts/default.html (hunk 1 above)
Comment (anagio) [reply _layouts/default.html:54]: ok algolia js is back with the search html file

**Pull request merged**